src/pages/UserProfile/UserProfile.js

```diff
@@ -0,0 +1,359 @@
+                        <Typography variant="body2" >
+                            Feb 2021 - Present · 1 yr 3 mos
+                        </Typography>
+                    </Box>
+                </Box>
+            </Box>
+
+        </LeftBoxConatiner>
+        <RightBoxConatiner>
+            <Box  bgcolor={ "Background.default"}
+                className="analytics-box" p={2}>
+                <Typography
+                    variant="h6"
+                    sx={{ color: "secondary" }}
+                    >
+                        People you may know
+                </Typography>
+                <Stack direction="column" spacing={2} justifyContent="space-between" >            
+                    <Stack direction="row" spacing={2} gap="20px" alignItems='center' >
+                        <Avatar alt="Remy Sharp" src="https://mui.com/static/images/avatar/1.jpg" sx={{ width: '4rem', height: '4rem' }}
+                        />
+                        <Stack direction="column" > 
+                            <Typography variant="h6" >
+                                Rohan jain
+                            </Typography>
+                            <Typography variant="body4" >
+                                UI/ UX Designer
+                            </Typography>       
+                        </Stack>
+                    </Stack>
+                    <Divider/>
+                    <Stack direction="row" spacing={2} gap="20px" alignItems='center' >
+                        <Avatar alt="Remy Sharp" src="https://mui.com/static/images/avatar/1.jpg" sx={{ width: '4rem', height: '4rem' }}
+                        />
+                        <Stack direction="column" > 
+                            <Typography variant="h6" >
+                                Rohan jain
+                            </Typography>
+                            <Typography variant="body4" >
+                                UI/ UX Designer
+                            </Typography>       
+                        </Stack>
+                    </Stack>
+                    <Divider/>
+
+                    <Stack direction="row" spacing={2} gap="20px" alignItems='center' >
+                        <Avatar alt="Remy Sharp" src="https://mui.com/static/images/avatar/1.jpg" sx={{ width: '4rem', height: '4rem' }}
+                        />
+                        <Stack direction="column" > 
+                            <Typography variant="h6" >
+                                Rohan jain
+                            </Typography>
+                            <Typography variant="body4" >
+                                UI/ UX Designer
+                            </Typography>       
+                        </Stack>
+                    </Stack>
+                    <Divider/>
+                   
+                </Stack>
+            </Box>
+        </RightBoxConatiner>
+        
+    </Stack>
+    </>
+  );
+};
+
+export default UserProfile;
```

Comment:
Consider breaking the code into smaller components. This file has around 360 lines of code. Also, remove unnecessary comments